### PR TITLE
For release v1.6.0

### PR DESCRIPTION
**Pull request by alsmk**
ip addresses are changed for qa, prod, staging, backup

### CODE REVIEW

File: infrastructure/known-hosts
Comment (Zangetsu101) [inline infrastructure/known-hosts:1]: Can we clean this file off of all the old IPs?

File: infrastructure/server-setup/inventory/staging.yml
Comment (Zangetsu101) [inline infrastructure/server-setup/inventory/staging.yml:1]: @rikukissa Do we really need to make a backup of the staging server? Does it bring any benefits to us?

Comment (rikukissa) [reply infrastructure/server-setup/inventory/staging.yml:1]: No, I don't think we do. As long as staging restores backups from prod we're solid

**Comment by rikukissa** [comment]
@Zangetsu101 Sorry I had to merge this in so I can get QA deployed. @alsmk can you please make a note of the things Tameem said. These values also need to be updated in develop

**Pull request merged**